app/models/work.rb

```diff
@@ -0,0 +1,21 @@
+class Work < ApplicationRecord
+  has_many :votes
+  has_many :users, through: :votes
+
+  validates :title, presence: true
+
+  def self.top
+    Work.all.max_by do |work| work.votes.length end
```

Comment:
For the sake of consistency, I would recommend curly braces and an explicit return:

```ruby
return Work.all.max_by { |work| work.votes.length }
```